plugin/web/theme/styles/theme.less

```diff
@@ -0,0 +1,48 @@
+ .ant-divider ,.ant-divider-horizontal{
```

Comment:
Can you please move these in to the styled components that use them. If you need them to modify, for example, ALL dividers, or ALL menu item icons, please do so inside the component that you are importing that antd component in to.

Reply:
Done , I realized that here I wont use divider , or radio buttons so I removed them for now , and the rest I added them to styled components as agreed

Reply:
https://github.com/eukarya-inc/reearth-plateauview/pull/29/commits/326aa3062b26c0e076d4cc56cf8ad8237c0cf1e2